app/src/main/java/be/chvp/nanoledger/data/LedgerRepository.kt

```diff
@@ -27,7 +27,7 @@ class LedgerRepository
         val accounts: LiveData<Set<String>> =
             transactions.map {
                 val result = HashSet<String>()
-                it.forEach { result.addAll(it.postings.map { it.account }) }
+                it.forEach { result.addAll(it.postings.map { it.account ?: "" }) }
```

Comment:
We are building the set of accounts for autocomplete here, so it seems cleaner to filter out the ones without an account instead of adding the empty string to that set.